### ISSUE TITLE
Build failling when the project is launching without problems

### ISSUE THREAD

**Issue by Org-gith** [renpy/renpy#4721]
Gone as far as renpy-7-nightly-2022-12-12 and it is not able to build it even though it can be launched, and was able to play it also without problems. Was not occurring on 7.5. This is the issue that occurs when building it:
![image](https://github.com/renpy/renpy/assets/78823063/da0ff4af-85e0-4418-b994-324b00700919)

Building "The Question" works without issues.
I'm expecting something wrong on my end but I fail to see something since it's launching and everything seems to work just fine.

**Comment by Org-gith** [comment]
I removed these lines from project.rpy:

```
            if wait:
                if p.wait():

                    if args and not self.is_writeable():
                        interface.error(_("Launching the project failed."), _("This may be because the project is not writeable."))
                    else:
                        interface.error(_("Launching the project failed."), _("Please ensure that your project launches normally before running this command."))
```

Just to see what will happen if nothing told it to stop, and this time it built successfully to the end, also ran the build it made and it seems to be working.

**Comment by renpytom** [comment]
So, this is waiting for the launch to finish, then checking its error code to figure out what's wrong. So you've gotten rid of the test for the error, not the error.

**Comment by renpytom** [comment]
Any chance I could try to build the project here to see if it fails for me?

**Comment by Org-gith** [comment]
Thank you, I sent it through email. This is a lite version compressed in Webp so that it doesn't take 7gigs but still launches and fails to build here on a fresh 7.6.1 install.

**Comment by renpytom** [comment]
So, the game has a lot of issues in it. The big one is that many of your files have every line indented by four spaces. This was never legal Ren'Py, but older versions did allow it. Ren'Py now correctly enforces that files can't be fully indented like this. 

There is also another issue with screens. 

There was also an issue where Ren'Py wouldn't report the errors when the game was being recompiled as part of the build process. That error has been fixed now, so if you use tomorrow's nightly fix build, you'll see all the errors your game has.

**Comment by renpytom** [comment]
For the record, the fix was 8ec26b7b66fa39e8085b9235cdf19bcfe8c3bba2 .

**Comment by Org-gith** [comment]
Awesome, I assumed some of the project stuff was going wrong, I was only aware of the indentation problem since I had tried a build somewhere between 7.5 and 7.6 in the past and it was reporting that but on 7.6.1 it wasn't displaying anymore. For the rest, I didn't know there was all of this, I'll fix them prompto.
Thanks for it, I close the issue then.
